<launch>
 
  <!-- Configure OpenNI in the right way. -->
  <node pkg="dynamic_reconfigure" type="dynparam" name="load_openni_params"
        args="load /camera/driver $(find rgbd_evaluator)/cfg/openni_params.yaml"/>
        
  <arg name="standalone" default="true"/>
  <arg if="$(arg standalone)" name="nodelet_args" value="standalone rgbd_evaluator/RgbdEvaluatorNodelet"/>
  <arg unless="$(arg standalone)" name="nodelet_args" value="load rgbd_evaluator/RgbdEvaluatorNodelet camera_nodelet_manager"/>
  
  <!-- Load RGBD Evaluator either standalone
       or into the same nodelet manager as OpenNI -->
  <node pkg="nodelet" type="nodelet" name="rgbd_evaluator" args="$(arg nodelet_args)" output="screen">
    <remap from="rgb_image" to="/camera/rgb/image_rect_color"/>
    <remap from="depth_image" to="/camera/depth_registered/image_rect"/>
    <remap from="camera_info" to="/camera/rgb/camera_info"/>
  </node>
  
  <!--
  <node pkg="rgbd_evaluator" type="rgbd_evaluator_node" name="rgbd_evaluator_node" output="screen">
    <remap from="rgb_image" to="/camera/rgb/image_rect_color"/>
    <remap from="depth_image" to="/camera/depth_registered/image_rect"/>
    <remap from="camera_info" to="/camera/rgb/camera_info"/>
  </node>
  -->

</launch>
